docs/v2_migration.md

```diff
@@ -10,6 +10,7 @@ Conan v2 in pull requests.
   * [Using Layout](#using-layout)
     * [With New Generators](#with-new-generators)
     * [With Multiple Build Helpers](#with-multiple-build-helpers)
+  * [New conf_info properties](#new-conf_info-properties)
```

Comment:
This should be here 🤔

Reply:
Looks like it was because of https://github.com/conan-io/conan-center-index/commit/9c774bfc60da85e5ae3be4f469f115ef9492ef68